cloudevents/spec.md

```diff
@@ -170,9 +170,11 @@ the available character set of all attributes such that case-sensitivity issues
 or clashes with the permissible character set for identifiers in common
 languages are prevented.
 
-CloudEvents attribute names MUST consist of lower-case letters ('a' to 'z') or
-digits ('0' to '9') from the ASCII character set. Attribute names SHOULD be
-descriptive and terse and SHOULD NOT exceed 20 characters in length.
+In order to maximize the likelihood of interoperability and portability across
+as many transport protocols and messaging formats, CloudEvents attribute names
```

Comment:
Add "as possible" before the comma? Otherwise "as many transport protocols and messaging formats" doesn't really work as a phrase :)

(Of course, having the requirement that the attributes can only consist of a-z and 0-9 *reduces* interoperability in other ways - if you're trying to convert another messaging format metadata *into* CloudEvents, and that other messaging format metadata has capital letters etc, then you're in a tricky position.)

Reply:
went with option 2 - thanks